dev/push-setup/Gemfile

```diff
@@ -0,0 +1,8 @@
+source 'https://rubygems.org'
+
```

Comment:
The setup script doesn't need it but yeah the user will want it to be installed so they can use knife to play with push jobs. I'll add it.